electron/legendary_utils/library.ts

```diff
@@ -105,8 +107,15 @@ export async function getLegendaryConfig(file: string) {
             keyImages,
             title,
             developer,
+            dlcItemList,
             customAttributes: { CloudSaveFolder, FolderName },
           } = metadata
+          dlcItemList ?
+            dlcItemList.map((v: { releaseInfo: any[] }) => {
```

Comment:
Since the map function is not returning anything, you could use a forEach instead.
Also changing the check condition to be a normal if/else would be better for readability I guess.